src/ScatterplotPanel.tsx

```diff
@@ -127,22 +127,15 @@ export const ScatterplotPanel: React.FC<Props> = ({ options, data, width, height
               d3.select(node).call(xAxis as any);
             }}
           />
-          <text className="axisLabel" transform={`translate(${chartWidth / 2}, ${chartHeight + margin.bottom - 10})`}>
-            {data.series.length > 1 ? data.series[0].name : ''}
-          </text>
           <g
             ref={node => {
               d3.select(node).call(yAxis as any);
             }}
           />
-          <text
-            className="axisLabel"
-            transform={`rotate(-90), translate(${-(chartHeight / 2)}, ${-(margin.left - 10)})`}
-          >
-            {data.series.length > 1 ? data.series[1].name : ''}
-          </text>
         </g>
       </svg>
+      <div className="axisLabel left-yaxis-label">{data.series.length > 1 ? data.series[0].name : ''}</div>
+      <div className={styles.bottomXaxisLabel}>{data.series.length > 1 ? data.series[1].name : ''}</div>
```

Comment:
Can we use the axisLabel class here as well, e.g. for the font size and apply additional css on top?

Reply:
good point, give me a moment

Reply:
done. You may change the lines too if you want.

Reply:
maybe we should keep position absolute and text-align-center in the css class because it makes otherwise no sense, what do you think?

Reply:
I am not sure where Grafana defines the properties of the `axisLabel` class. As long as position and text-center only apply to the X axis, it might be better to keep it in the css class.

Reply:
axisLabel is defined here: https://github.com/grafana/grafana/blob/master/public/sass/components/_panel_graph.scss#L333

The class already contains position and text-center, so leave it like it is.